hstream-http-server/src/HStream/HTTP/Server/Query.hs

```diff
@@ -69,10 +69,16 @@ hstreamQueryToQueryBO (HSP.Query queryId (HSP.Info sqlStatement createdTime) (HS
 hstreamQueryNameIs :: T.Text -> HSP.Query -> Bool
 hstreamQueryNameIs name (HSP.Query queryId _ _) = (cbytesToText queryId) == name
 
--- removeQueryHandler :: HS.LDClient -> String -> Handler Bool
--- removeQueryHandler ldClient name = do
---   liftIO $ removeQuery ldClient (mkQueryName $ ZDC.pack name)
---   return True
+removeQueryHandler :: HS.LDClient -> Maybe ZK.ZHandle -> String -> Handler Bool
+removeQueryHandler ldClient zkHandle name = do
+  liftIO $ do
+    queries <- HSP.withMaybeZHandle zkHandle HSP.getQueries
+    case find (hstreamQueryNameIs (T.pack name)) queries of
+      Just (HSP.Query queryId _ (HSP.Status status _)) -> do
+        catch
+          ((HSP.withMaybeZHandle zkHandle $ HSP.removeQuery queryId) >> return True)
+          (\(e :: SomeException) -> return False)
```

Comment:
No need to `find` if you use `catch` to capture the exception.

Reply:
ok. Let me see.

Reply:
done